
Currently Loaded Modules:
  1) Anaconda3/5.0.1-fasrc02   3) intel/17.0.4-fasrc01
  2) python/3.6.3-fasrc02      4) impi/2017.2.174-fasrc01

 

Traceback (most recent call last):
  File "/n/home04/jzlu/codes/ml-ph/ALLEGRO_ANALYZER/start.py", line 37, in <module>
    bze_points = begin_computation(user_input_settings)
  File "/n/home04/jzlu/codes/ml-ph/ALLEGRO_ANALYZER/__begin_computation.py", line 54, in begin_computation
    config = Configuration(BASE_ROOT)
  File "/n/home04/jzlu/codes/ml-ph/ALLEGRO_ANALYZER/__class_Configuration.py", line 23, in __init__
    poscars = self.import_init_poscars()
  File "/n/home04/jzlu/codes/ml-ph/ALLEGRO_ANALYZER/__class_Configuration.py", line 45, in import_init_poscars
    print('Fixed layer set to POSCAR named ' + poscar_names[0])
IndexError: list index out of range
